mix.exs

```diff
@@ -33,6 +33,7 @@ defmodule Kino.MixProject do
     [
       {:table, "~> 0.1.2"},
       {:nx, "~> 0.1", optional: true},
+      {:explorer, "~> 0.4.0", optional: true},
```

Comment:
This should point to main for now, because we don't have `DF.describe` in 0.4.0, and this way we don't forget to bump to `0.5.0` once it's out.

Reply:
The dependency on Explorer is temporary... we should talk how we want to handle this via Table/Explorer. :) There are three functionalities:

1. Aggregations
2. Summaries
3. Sorting

We can have a convo later today!